src/mSupplyMobileApp.js

```diff
@@ -209,10 +200,7 @@ class MSupplyMobileAppContainer extends React.Component {
 
   render() {
     const {
-      finaliseItem,
       currentUser,
-      finaliseModalOpen,
-      closeFinaliseModal,
```

Comment:
finalise modal is now connected straight to redux, so fetches these props from there. Less root rendering woo!